src/world/common/SetGameStatusUnk84_1.inc.c

```diff
@@ -0,0 +1,6 @@
+#include "common.h"
+
+static ApiStatus SetGameStatusUnk84_1(ScriptInstance* script, s32 isInitialCall) {
+    GAME_STATUS->unk_84 = 1;
+    return ApiStatus_BLOCK;
```

Comment:
hm, interesting that this always block; won't this cause an infloop if called?

Reply:
This either may not be an ApiStatus or the context in which it's used prevents it from being an infinite loop. @nanaian may have more input

Reply:
They definitely do this sometimes. It won't create an infinite loop, just make the script never continue.